src/codegen/protos/session.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
```

Comment:
Because we are duplicating the `session.proto` codegen in nidaqmx-python, nimi-python, and measurementlink-python, we must use exactly the same version of `session.proto` for all three, or else users will get errors when using nidaqmx-python with either nimi-python or measurementlink-python. 

For now, the solution is to copy `session.proto` from nimi-python or measurementlink-python.

As a manual test, run `poetry run pip install ni-measurementlink-service` and `poetry run pip install nidcpower`, then `poetry run python`, `import nidaqmx.stubs.nidaqmx_pb2_grpc`, `import ni_measurementlink_service`, and `import nidcpower.nidcpower_pb2_grpc`.

The long term solution will probably involve packaging shared protos in separate Python modules and is tracked by AB#2250775: session.proto versioning is extremely brittle in Python and **will** break when MeasurementLink or nimi-python upgrades

Reply:
Updated the `session.proto` from nimi-python.